src/main/scala/org/broadinstitute/dsde/vault/DmClientService.scala

```diff
@@ -34,10 +37,11 @@ object DmClientService {
   case class DMLookupEntity(entityType: String, attributeName: String, attributeValue: String)
   case class DMLookupResolved(result: EntitySearchResult)
 
+
   def props(requestContext: RequestContext): Props = Props(new DmClientService(requestContext))
 }
 
-case class DmClientService(requestContext: RequestContext) extends Actor {
+case class DmClientService(requestContext: RequestContext) extends Actor with DefaultJsonProtocol{
```

Comment:
The style I've seen for making implicits available has generally been similar to line 46 below, with an inner import:

`import DefaultJsonProtocol._`